landing-page/src/App.css

```
@@ -97,3 +121,651 @@
+  align-self: stretch;
+  flex-grow: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: stretch;
+  padding: 0;
+  border-radius: 16px;
+  -webkit-backdrop-filter: blur(8px);
+  backdrop-filter: blur(8px);
+  background-color: rgba(255, 255, 255, 0.01);
+  flex: 1;
```

Biome (2.1.2)

[error] Unexpected shorthand property flex after flex-grow



(lint/suspicious/noShorthandPropertyOverrides)